doc/DeveloperGuide.md

```diff
@@ -87,6 +87,42 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+#### Use case: Rename tag
+
+**Precondition**
+
+All process working fine. 
+
+**Guarantees**
+
+The tag is renamed.
+
+**MSS**
+
+1. User enters command 'rename tag'
```

Comment:
no need to specify the command here. command you provide is implementation specific.